app/views/layouts/application.html.erb

```diff
@@ -115,5 +115,10 @@
   <%= javascript_include_tag :attachinary %>
   <%= yield :charts_data %>
   <%= cloudinary_js_config %>
+
+  <script type="text/javascript">
```

Comment:
It should be already be done automatically by this: https://github.com/Codeminer42/cm42-central/blob/master/app/views/layouts/application.html.erb#L114, weird o_O